src/Apps/Order/Components/TransactionSummary.tsx

```diff
@@ -0,0 +1,100 @@
+import { Serif, space } from "@artsy/palette"
+import React from "react"
+import { media } from "styled-bootstrap-grid"
+import { BorderBox, Box } from "Styleguide/Elements/Box"
+import { Flex, FlexProps } from "Styleguide/Elements/Flex"
+import { Image } from "Styleguide/Elements/Image"
+import { Separator } from "Styleguide/Elements/Separator"
+import { Spacer } from "Styleguide/Elements/Spacer"
+import styled from "../../../../node_modules/styled-components"
+
+const SummaryContainer = styled(Flex)`
+  margin: ${space(2)}px;
+  ${media.sm`
+  margin: ${space(3)}px;
+`};
+`
+
+export interface TransactionSummaryProps extends FlexProps {
+  price: string
+  shipping: string
+  tax: string
+  total: string
+  artistName: string
+  artworkName: string
+  sellerName: string
+  artworkLocation: string
+  imageURL: string
+}
+
+export const TransactionSummary: React.SFC<TransactionSummaryProps> = ({
+  price,
+  shipping,
+  tax,
+  total,
+  artistName,
+  artworkName,
+  sellerName,
+  artworkLocation,
+  imageURL,
+  ...others
+}) => {
+  return (
+    <BorderBox p={0} flexDirection="column" {...others}>
```

Comment:
Just noticed your comment about the separator width and StackableBorderBox. This kind of thing was intended to be achieved was like this:

```js
<Flex flexDirection="column">
  <StackableBorderBox>{...}</StackableBorderBox>
  <StackableBorderBox>{...}</StackableBorderBox>
</Flex>
```

That way you don't need explicit separators. The 'Stackable' aspect of the box means that the borders blend into one seamlessly. This would also take care of the padding, so you wouldn't need the SummaryContainer element.

What you've done works fine though, I see no reason to change it right now except for the sake of consistency with other components.

Reply:
Oh! That makes _so_ much more sense, thank you! I think it's worth it to be consistent with the other components – I'll update the PR.